ragna/assistants/_ai21labs.py

```diff
@@ -0,0 +1,80 @@
+from typing import AsyncIterator, cast
+
+from ragna.core import RagnaException, Source
+
+from ._api import ApiAssistant
+
+
+class Ai21LabsAssistant(ApiAssistant):
+    _API_KEY_ENV_VAR = "AI21_API_KEY"
+    _MODEL_TYPE: str
+    _CONTEXT_SIZE: int = 8_192
```

Comment:
Could you include a link on where to find this information?

Reply:
Done

Reply:
Perfect, thanks!